tests/unit_tests/test_wave_06.py

```diff
@@ -260,12 +250,9 @@ def test_swap_best_by_category_no_other_match_is_false():
         my_priority="Electronics",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    
+    assert not result
```

Comment:
We should explicitly check that `result` is equal to `False` instead of just checking that `result` is a falsey value

```suggestion
    assert result == False
```